packages/riipen-ui/src/components/RadioGroup.jsx

```diff
@@ -39,12 +40,13 @@ const RadioGroup = ({
   return (
     <>
       <fieldset>
-        {(label || hint) && (
+        {(label || hint || suffix) && (
           <InputLabel
             hint={hint}
             htmlFor={other.name}
             required={required}
             {...labelProps}
+            suffix={suffix}
```

Comment:
i believe we should swap order here, based on what im reading, i think label props should overwrite any existing props